### ISSUE TITLE
NPE on Java 15

### ISSUE THREAD

**Issue by boris-petrov** [w3c/css-validator#294]
I updated our project to use Java 15 and I started getting a bunch of:

```
Error: CSS: “top”: Cannot invoke "org.w3c.css.values.CssValue.getType()" because "this.val1" is null.
```

This seems to mostly/only happen on [FullCalendar](https://github.com/fullcalendar/fullcalendar)'s CSS on, for example, this code (on the `calc` line exacty):

```css
.fc-v-event:not(.fc-event-selected) .fc-event-resizer-start {
  top: -4px;
  top: calc(var(--fc-event-resizer-thickness, 8px) / -2);
}
```

**Comment by ylafon** [comment]
This is related to improper parsing of variables in complex values (like functions). Indeed a bug

**Comment by ylafon** [comment]
fixed by #316
